src/Lorentz/Contracts/GameDAO.hs

```diff
@@ -203,13 +225,34 @@ decisionLambda = do
     setField #ceAcceptedProposalsNum
     stSetField #sExtra
 
-config :: DAO.Config GameDaoContractExtra GameDaoProposalMetadata
-config = DAO.defaultConfig
+extraEntrypoints
+  :: forall s store. (DAO.StorageC store GameDaoContractExtra GameDaoProposalMetadata)
+  => GameDaoExtraInterface : store : s :-> List Operation : store : s
+extraEntrypoints =
+  entryCase (Proxy @GameDaoCustomEntrypointsKind)
+    ( #cDefault /-> do
+        doc $ DDescription "Entrypoint used for transferring money to the contract."
+        nil
+    , #cResetProposalsCounter /-> do
+        doc $ DDescription "An entrypoint that performs something smart (or not)."
+        stGetField #sExtra
+        push 0
+        setField #ceAcceptedProposalsNum
+        stSetField #sExtra
+        nil
+    )
+
+config :: DAO.Config GameDaoContractExtra GameDaoProposalMetadata GameDaoExtraInterface
+config = DAO.Config
```

Comment:
Does it mean we can't use `defaultConfig` anymore (as we previously did)?

Reply:
We can, I just find this pattern not good because one can forget to initialize some fields (e.g. compiler didn't warn me that I forgot to add an implementation for new entrypoints). And values that are inherited from default config (like voting periods) are redefined here anyway.

Mm, actually nothing stops me from putting this change in a separate commit.